app/src/features/docs/components/DocsSearch.tsx

```diff
@@ -0,0 +1,396 @@
+          ))}
+        </Box>
+      )}
+    </Box>
+  );
+
+  return (
+    <Container maxWidth="lg" sx={{ py: 6 }}>
+      {/* Header - similar to docs.rs */}
+      <Box textAlign="center" mb={6}>
+        <Box 
+          component="img" 
+          src="/sway_logo.png" 
+          alt="Sway"
+          sx={{ 
+            height: 80,
+            mb: 3,
+            // Keep natural colors for the palm tree
+          }}
+        />
+        <Typography 
+          variant="h2" 
+          component="h1" 
+          sx={{ 
+            fontWeight: 700,
+            fontSize: { xs: '2rem', md: '2.8rem' },
+            mb: 3,
+            color: 'white'
+          }}
+        >
+          docs.forc.pub
+        </Typography>
+      </Box>
+
+      {/* Search Bar */}
+      <Box display="flex" justifyContent="center" mb={6}>
+        <TextField
+          variant="outlined"
+          placeholder="Type 'S' or '/' to search"
+          value={searchQuery}
+          onChange={handleSearchChange}
+          sx={{ 
+            width: { xs: '100%', sm: '500px' },
+            '& .MuiOutlinedInput-root': {
+              backgroundColor: 'white',
+              borderRadius: 1,
+              fontSize: '1.1rem',
+              '& fieldset': {
+                borderColor: '#ccc',
+              },
+              '&:hover fieldset': {
+                borderColor: '#999',
+              },
+              '&.Mui-focused fieldset': {
+                borderColor: '#007acc',
+              },
+            },
+            '& .MuiInputBase-input': {
+              py: 1.5,
+              color: 'black'
+            }
+          }}
+          InputProps={{
+            startAdornment: (
+              <InputAdornment position="start">
+                <SearchIcon sx={{ color: '#666' }} />
+              </InputAdornment>
+            ),
+          }}
+        />
+      </Box>
+
+
+      {error && (
+        <Alert severity="error" sx={{ mb: 3, maxWidth: '500px', mx: 'auto' }}>
+          {error}
+        </Alert>
+      )}
+
+      {/* Search Results */}
+      {searchQuery && searchResults.length > 0 && (
+        <Box maxWidth="800px" mx="auto">
+          <Typography variant="h5" gutterBottom sx={{ fontWeight: 600, mb: 3 }}>
+            Search Results
+          </Typography>
+          {renderSearchResults(searchResults)}
+        </Box>
+      )}
+
+      {/* Recent Releases - only show when not searching */}
+      {!searchQuery && recentPackages.recentlyUpdated.length > 0 && (
+        <Box maxWidth="800px" mx="auto">
+          {renderRecentReleases(
+            recentPackages.recentlyUpdated
+              .filter(pkg => pkg.docsIpfsUrl !== null)
+              .sort((a, b) => {
+                // Sort by most recent update time (updatedAt), fallback to createdAt
+                const aTime = new Date(a.updatedAt || a.createdAt || '0').getTime();
+                const bTime = new Date(b.updatedAt || b.createdAt || '0').getTime();
```

Comment:
nit: not that it will make a difference - but there is a difference between `new Date(0)` and `new Date('0')`.
You can check in browser:
```
> new Date('0')
Sat Jan 01 2000 00:00:00 GMT+1300 (New Zealand Daylight Time)
> new Date(0)
Thu Jan 01 1970 12:00:00 GMT+1200 (New Zealand Standard Time)
```
Based on this; im assuming you may want to use `new Date(0)` (`new Date(a.updatedAt || a.createdAt || 0)`) - which is the general default (zero) date.